docs/data/material/components/material-icons/SearchIcons.js

```diff
@@ -518,29 +518,31 @@ const searchIndex = new FlexSearchIndex({
   tokenize: 'full',
 });
 
+const nameCleanRegex = /(Outlined|TwoTone|Rounded|Sharp)$/;
```

Comment:
We could even get rid of it altogether and rely purely on string manipulation. e.g.

```tsx
let theme = 'Filled';
let name = importName;

for (const currentTheme of ['Outlined', 'Rounded', 'Two tone', 'Sharp']) {
  if (importName.endsWith(currentTheme)) {
    theme = currentTheme;
    name = importName.slice(0, -currentTheme.length);
    break;
  }
}
```

Reply:
I couldn't measure a performance difference, but why not, going for this, just that "Two tone" != "TwoTone".